frontend/src/i18n/locales/de/frequent.json

```
@@ -0,0 +1,124 @@
+{
+  "Whoops! This page doesn't exist": "Huch! Diese Seite existiert nicht",
+  "Head back <1>home</1>.": "Zurück <1>nach Hause</1>.",
+  "Cancel": "Abbrechen",
+  "Something went wrong with cluster {{ cluster }}": "Mit dem Cluster ist etwas schief gelaufen {{ cluster }}",
+  "Choose another cluster": "Wählen Sie einen anderen Cluster",
+  "Skip to main content": "Zum Hauptinhalt springen",
+  "Notifications": "Nachrichten",
+  "Date": "Datum",
+  "Name": "Name",
+  "Server": "Server",
+  "Apply": "Anwenden",
+  "Delete": "Löschen",
+  "Custom value": "Benutzerdefinierter Wert",
+  "General": "Allgemein",
+  "Language": "Sprache",
+  "Theme": "Design",
+  "Settings": "Einstellungen",
+  "light theme": "helles Theme",
+  "dark theme": "dunkles Theme",
+  "Change theme": "Theme ändern",
+  "Log out": "Abmelden",
+  "(No token set up)": "(Kein Token eingerichtet)",
+  "Account of current user": "Konto des aktuellen Benutzers",
+  "Appbar Tools": "Appbar-Tools",
+  "show more": "mehr anzeigen",
+  "Version": "Version",
+  "Git Commit": "Git Commit",
+  "Try Again": "Erneut versuchen",
+  "Back": "Zurück",
+  "Next": "Weiter",
+  "Finish": "Beenden",
+  "Offline": "Offline",
+  "No": "Nein",
+  "Yes": "Ja",
+  "Toggle fullscreen": "Vollbild ein/aus",
+  "Close": "Schließen",
+  "Find": "finden",
+  "All namespaces": "Alle Namespaces",
+  "Type": "Typ",
+  "Reason": "Ereignis",
+  "From": "Von",
+  "Message": "Nachricht",
+  "Age": "Alter",
+  "Skip": "Überspringen",
+  "Read more": "Mehr lesen",
+  "Dismiss": "Ablehnen",
```

```suggestion
  "Dismiss": "Schließen",
```
This is an interesting one. I guess "Schließen" would work best here since I'm guessing this is the label on a button to dismiss a dialog. Depends on the context though. "Ablehnen" means "Deny" and will most likely look funny here.

---

It's a Close button on a 'snackbar' message like popup thingo. Yeah ok, Schließen sounds better. "Be gone dastardly popup!".